PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java

```diff
@@ -133,6 +100,23 @@ public static SignInWindow go(AuthMethodDetails authMethodDetails, Project proje
         return null;
     }
 
+    public AuthType getData() {
```

Comment:
it should return `AuthConfiguration` which is just same as maven.
let's refactor this part in the future.